target/scripts/startup/setup-stack.sh

```diff
@@ -1512,3 +1512,13 @@ function _setup_fail2ban
     echo -e "[Init]\nblocktype = DROP" > /etc/fail2ban/action.d/iptables-common.local
   fi
 }
+
+function _setup_dnsbl_disable
+{
+  _notify 'task' 'Disabling postfix DNS block list (zen.spamhaus.org)'
+  sedfile -i '/^smtpd_recipient_restrictions = / s/, reject_rbl_client zen.spamhaus.org//' /etc/postfix/main.cf
```

Comment:
Will this fail similar to the current TLS logic with `sedfile` during a container restart? (_as opposed to recreating the container from scratch_)

May want to just use `sed` or have `sedfile` not fail if no match is found to alter?

Reply:
Yes, this behaves the same as the TLS logic at the moment. I am unsure how we should handle this in general.
Errors on a container restart are nasty. On the other I think there is value in getting to know when this replacement does not take place for some reason on regular container starts.
We can open a separate issue to discuss/vote how to proceed with `sedfile` usage outside of `Dockerfile`.